bubbles/commands/periodic/get_in_progress_posts.py

```diff
@@ -1,21 +1,19 @@
+from blossom_wrapper import BlossomStatus
 from bubbles.config import blossom, app, ENABLE_BLOSSOM
 
+from requests.exceptions import HTTPError
+
 CHANNEL = "transcription_check"
 
 
 def get_in_progress_callback():
     if not ENABLE_BLOSSOM:
         return
 
-    result = blossom.get("submission/in_progress", params={"source": "reddit"})
-
-    if not str(result.status_code).startswith("2"):
-        msg = f"Got a weird response; Blossom returned a {result.status_code}."
-
-    else:
-        result = result.json()
 
-        if len(result) == 0:
+    try:
+        result = blossom.get("submission/in_progress", params={"source": "reddit"})
+        if len(result.data) == 0:
```

Comment:
We're not actually assigning the data to anything in the wrapper -- this line doesn't look like it will do anything.